File: examples/android-gradle/src/main/java/dev/aleksrychkov/example/MethodHook.java
package dev.aleksrychkov.example;

public class MethodHook {

    /**
     * @noinspection unused
     */
    public static void start(String runtimeClazz, String clazz, String method) {
        System.out.println("MethodHook::start::" + runtimeClazz + "::" + clazz + "::" + method);
    }

    /**
     * @noinspection unused
     */
    public static void end(String runtimeClazz, String clazz, String method) {
        System.out.println("MethodHook::end::" + runtimeClazz + "::" + clazz + "::" + method);
    }
}

File: examples/android-gradle-kts/src/main/kotlin/dev/aleksrychkov/example/MethodHook.kt
package dev.aleksrychkov.example

object MethodHook {

    @JvmStatic
    fun start(runtimeClazz: String, clazz: String, method: String) {
        println("MethodHook::start::$runtimeClazz::$clazz::$method")
    }

    @JvmStatic
    fun end(runtimeClazz: String, clazz: String, method: String) {
        println("MethodHook::end::$runtimeClazz::$clazz::$method")
    }
}
